.readthedocs.yml

```diff
@@ -0,0 +1,16 @@
+# .readthedocs.yml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+python:
+  version: 3.6
+  install:
+    - requirements: requirements.txt
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  builder: html
+  configuration: docs/conf.py
+  fail_on_warning: false
```

Comment:
please set this to `true`. Its not easy to access the RTD build logs, especially for those that aren't maintaining the RTD project, just to see what went wrong. We should also augment the contributing guidelines about locally building the doc changes before committing them.

Reply:
I added a favicon.ico (along with the GIMP file used to create it) to ./docs/_static as that was the only reason the build was failing.